app/lib/routes/settings.dart

```diff
@@ -119,6 +229,25 @@ class BellSoundControl extends StatelessWidget {
   }
 }
 
+class ShareBikeControl extends StatelessWidget {
+  const ShareBikeControl(this.bike, {super.key});
+
+  final Bike bike;
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      onPressed: () {
+        showModalBottomSheet(
+          context: context,
+          builder: (context) => ShareSettings(bike: bike),
+        );
+      },
+      child: const Text('Manage sharing'),
```

Comment:
Maybe it's nice to add an icon in front of this

Reply:
![image](https://user-images.githubusercontent.com/15117158/184347800-b04746bf-7b98-49eb-ae9d-b37c5becb827.png)

Reply:
![image](https://user-images.githubusercontent.com/15117158/184348079-193b2e89-1e42-4577-a0a5-f3238129bae7.png)